services/distribution/src/main/resources/main-config/v2/risk-calculation-parameters-1.15.yaml

```diff
@@ -0,0 +1,94 @@
+#----------------------------------------------------------------------
+# This is the Corona Warn App source configuration file.
+# ----------------------------------------------------------------------
+#
+# This configuration file will be fetched by the mobile app in order to function.
+#
+# Change this file with caution!
+
+
+minutes-at-attenuation-filters: [] # no attenuation filters
+
+trl-filters: [] # no TRL filters
+
+minutes-at-attenuation-weights:
+  -
+    attenuation-range:
+      min: 0
+      max: 55
+      max-exclusive: true
+    weight: 1.0
+  -
+    attenuation-range:
+      min: 55
+      max: 63
+      max-exclusive: true
+    weight: 0.5
+  -
+    attenuation-range:
+      min: 63
+      max: 73
+      max-exclusive: true
+    weight: 0.3
```

Comment:
@mlenkeit / @ioangut - is the yaml correct? does it work properly in regards to the array?

Reply:
I think yes.  Max reviewed it also and add some changes.